client/src/document/molecules/_notecards.scss

```diff
@@ -44,7 +46,8 @@
   }
 
   h3,
-  h4 {
+  h4,
+  #note {
```

Comment:
Perhaps I've missed something but we should never have an `id="note"`. 
I don't see any in the content.

Reply:
They do exist! Here is one for reference. 

https://developer.mozilla.org/en-US/docs/Glossary/HTML5

<img width="1280" alt="Screen Shot 2021-08-04 at 8 40 41 AM" src="https://user-images.githubusercontent.com/48612525/128211038-982a6f4e-6c2c-4daa-ab19-5e05a6e07ef0.png">

Reply:
Needless to say, since all the `.notecard h4`s are going away anyways. I just removed the changes to scss that I made to mimic the styles of the `<h4>` with the ::before. We can come back and adjust with a class/id selector down the line if we wish to still utilize the background-image for the little notecard icon etc.